build-libssp.sh

```diff
@@ -37,36 +37,18 @@ export PATH="$PREFIX/bin:$PATH"
 : ${ARCHS:=${TOOLCHAIN_ARCHS-i686 x86_64 armv7 aarch64}}
 
 if [ ! -d libssp ]; then
-    svn checkout -q https://github.com/gcc-mirror/gcc/tags/releases/gcc-7.3.0/libssp
+    svn checkout -q svn://gcc.gnu.org/svn/gcc/tags/gcc_9_2_0_release/libssp 
+    patch -p1 < 0002-allow-building-outside-of-the-gcc-tree.patch
```

Comment:
Instead of manually listing the patches here, I'd suggest something like this:
```
for i in patches/libssp/*.patch; do
    patch -p1 < $i
done
```

Reply:
OK